File: an2_sem4/MPP/Homework/sapt14/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/GetAllSpectacoleResponse.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;


public class GetAllSpectacoleResponse extends SpectacoleResponse {
    public GetAllSpectacoleResponse(Spectacol[] spectacole) {
        super(spectacole);
    }
}

File: an2_sem4/MPP/Homework/sapt11/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/GetAllSpectacoleResponse.cs
using FestivalSellpoint.Domain;
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    internal class GetAllSpectacoleResponse : SpectacoleResponse
    {
        public GetAllSpectacoleResponse(Spectacol[] spectacole) : base(spectacole) { }  
    }
}
